plugin/helpers_plugin_test.go

```diff
@@ -0,0 +1,102 @@
+package plugin_test
```

Comment:
@RajatVaryani Would you please add a licence header similar to the one in `plugin/helpers_plugin.go` to fix the build?